argo_workflow_tools/workflow_status.py

```diff
@@ -7,11 +7,15 @@ class WorkflowStatus(Enum):
     Failed = "Failed"
     Canceled = "Canceled"
     Suspended = "Suspended"
+    Created = "Created"
 
     @classmethod
     def value_of(cls, value):
+        """Return the enum value of the given value. if no value is found, return None."""
         for k, v in cls.__members__.items():
             if k == value:
                 return v
         else:
+            if value is None:
```

Comment:
@8ball030 please move this before the for-loop above for early returning (super not critical :) )

Reply:
lets see if we get any feedback from the tests first? Might have a few changes?